### PR TITLE
Rename redundancy action plan to redundancy tool

### PR DESCRIPTION
**Pull request by tomas-stefano**
* #6278 in Target process.
* Rename the existing URL https://www.moneyadviceservice.org.uk/en/action-plans/redundancy
to https://www.moneyadviceservice.org.uk/en/tools/redundancy-pay-calculator

### CODE REVIEW

File: spec/requests/tool_integration/action_plans_spec.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
-RSpec.describe 'RIO', type: :request, features: [:action_plans] do
+RSpec.describe 'Redundancy pay calculator', type: :request, features: [:action_plans] do
```
Comment (tomas-stefano) [inline spec/requests/tool_integration/action_plans_spec.rb]: Be carefull with copy and paste :smile:

**Comment by asmega** [comment]
:+1:

**Comment by darokel** [comment]
:+1:

**Comment by Tr4pSt3R** [comment]
:+1:

**Pull request merged**